crates/nu-utils/src/sample_config/default_env.nu

```diff
@@ -87,3 +87,13 @@ $env.NU_PLUGIN_DIRS = [
 
 # To add entries to PATH (on Windows you might use Path), you can use the following pattern:
 # $env.PATH = ($env.PATH | split row (char esep) | prepend '/some/path')
+# To add multiple paths to PATH this may be simpler:
```

Comment:
I think this can be worded a little better.
```suggestion
# An alternate way to add entries to $env.PATH is to use the custom command `path add`
# which is built into the nushell stdlib:
```